README.md

```diff
@@ -100,7 +100,9 @@ have to manually set the workspace to the redis URI.
 ## Usage
 
 ```
-$ manticore ./path/to/binary  # runs, and creates a mcore_* directory with analysis results
+$ manticore ./path/to/binary        # runs, and creates a mcore_* directory with analysis results
+$ manticore ./path/to/binary ab cd  # use concrete strings "ab", "cd" as argv
+$ manticore ./path/to/binary ++ ++  # use two symbolic strings of length two as argv
```

Comment:
"as argv" implies as the entire argv array. How about "Use symbolic strings of length two for `argv[1]` and `argv[2]`?" or "Use two symbolic strings of length two as program arguments". (Ditto for above)